pkg/evaluators/authorization/kubernetes_authz.go

```diff
@@ -63,26 +63,61 @@ func (k *KubernetesAuthz) Call(pipeline auth.AuthPipeline, ctx gocontext.Context
 	}
 
 	authJSON := pipeline.GetAuthorizationJSON()
-	jsonValueToStr := func(value json.JSONValue) string {
-		return fmt.Sprintf("%s", value.ResolveFor(authJSON))
+	jsonValueToStr := func(value expressions.Value) (string, error) {
+		if value == nil {
+			return "", nil
```

Comment:
Ugh... Now that it is a pointer, we could get `nil` indeed... I don't know whether all these cases are now covered, hard to tell what can or not be `nil` in those specs